GreenhouseNode/GreenhouseNode/links/CO2Handler.c

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
I thought you were only working on temperature and humidity sensors, why is this here?